electron-src/index.ts

```diff
@@ -94,6 +94,10 @@ ipcMain.on("front error", (event, errorTitle, errorMessage, data) => {
   mainWindow?.webContents.send("render error", errorTitle, errorMessage, data);
 });
 
+ipcMain.on("fetch error", (event, errorTitle, errorMessage, data) => {
+  mainWindow?.webContents.send("actualization error", errorTitle, errorMessage, data);
+});
+
```

Comment:
i guess we could optimize these two handlers. Why do we need to create new `fetch error` handler